### DISCUSSION TITLE
가상디바이스 화면에서 React 초기화면 (Welcome to React...)으로 바뀌지 않습니다.

### DISCUSSION THREAD

**Discussion by Zooplay07** [velopert/dealing-with-react-native#26] [Q&A]
아래 화면까지는 잘 진행되었습니다.
헌데..
React 초기 화면이 나오지 않고 계속 아래 화면만 나옵니다.
어떻게 해야 초기 Welcom to React - step one... 화면이 나오게 되나요?

![image](https://user-images.githubusercontent.com/105140149/167281344-e02507cf-cc86-475f-8c64-a99ee1f6160d.png)

**Comment by velopert** [comment]
프로젝트에서 `yarn android` 명령어를 실행하신 상태일까요?
실행 후 오류가 발생했다면 오류를 복사해서 여기에 남겨주세요 ~
